### ISSUE TITLE
Where is the old Readme.md with the overview chart on how to use the tool?

### ISSUE THREAD

**Issue by koopa990** [Anjok07/ultimatevocalremovergui#377]
Did you take it out permanently?

**Comment by Anjok07** [comment]
> Did you take it out permanently?

The chart was replaced by "Help Hints". Right-click anywhere in the application and select "Enable Help Hints". From there, you will get detailed information about each option when you hover over them.

**Comment by koopa990** [comment]
> > Did you take it out permanently?
> 
> The chart was replaced by "Help Hints". Right-click anywhere in the application and select "Enable Help Hints". From there, you will get detailed information about each option when you hover over them.

Ah, thank you. That is a nice addition. 
And I suppose the UVR does not have a community group?

**Comment by Anjok07** [comment]
> > > Did you take it out permanently?
> > 
> > The chart was replaced by "Help Hints". Right-click anywhere in the application and select "Enable Help Hints". From there, you will get detailed information about each option when you hover over them.
> 
> Ah, thank you. That is a nice addition. 
> And I suppose the UVR does not have a community group?

Yes! The Audio Separation Community on Discord is a great resource for all things UVR and source separation related.

Link: https://discord.gg/J4xn3QbB

**Comment by koopa990** [comment]
> > > > Did you take it out permanently?
> > > 
> > > 
> > > The chart was replaced by "Help Hints". Right-click anywhere in the application and select "Enable Help Hints". From there, you will get detailed information about each option when you hover over them.
> > 
> > 
> > Ah, thank you. That is a nice addition.
> > And I suppose the UVR does not have a community group?
> 
> It does! The Audio Separation Community on Discord is a great resource for all things UVR and source separation related.
> 
> Link: https://discord.gg/J4xn3QbB

sweet! thank you!

**Comment by 0xdevalias** [comment]
> The Audio Separation Community on Discord is a great resource for all things UVR and source separation related.

Updated link I found in another issue on this repo when I searched "discord":

> Join the dedicated Discord server for all the info: https://discord.gg/zZ9CbEn5HE
> 
> _Originally posted by @thenormal in https://github.com/Anjok07/ultimatevocalremovergui/issues/344#issuecomment-1527894115_
